src/github.com/getlantern/autoupdate-server/server/asset.go

```diff
@@ -46,6 +49,10 @@ func downloadAsset(uri string) (localfile string, err error) {
 			return "", fmt.Errorf("Expecting 200 OK, got: %s", res.Status)
 		}
 
+		if res.Body != nil {
+			defer res.Body.Close()
+		}
+
```

Comment:
If http.Get doesn't return an error, do we need to check the response body for nil here?